packages/@uppy/image-editor/src/Editor.jsx

```diff
@@ -14,10 +17,29 @@ export default class Editor extends Component {
 
   componentDidMount () {
     const { opts, storeCropperInstance } = this.props
+    let prevCropboxData = null
     this.cropper = new Cropper(
       this.imgElement,
-      opts.cropperOptions,
+      {
+        ...opts.cropperOptions,
+        cropstart: () => {
+          prevCropboxData = this.cropper.getCropBoxData()
+        },
+        cropend: (event) => {
```

Comment:
Could technically be a breaking change if users passed those options themselves. Could perhaps be okay to not release it as such, but it should be documented at least on the website I think.

Reply:
Oh, right, thanks for noting. We can add these events in a way that won't overwrite user's options, I'll do it.